pkgs/development/tools/abuild/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, pkg-config
+, openssl
+, zlib
+, busybox
+}:
+
+stdenv.mkDerivation rec {
+  pname = "abuild";
+  version = "3.7.0";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.alpinelinux.org";
+    owner = "alpine";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1xsik9hyzzq861bi922sb5r8c6r4wpnpxz5kd30i9f20vvfpp5jx";
+  };
+
+  buildInputs = [
+    openssl
+    zlib
+    busybox
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  patchPhase = ''
+    substituteInPlace ./Makefile \
+      --replace '-Werror' '-Wno-error' \
+      --replace 'chmod' 'echo'
+  '';
+
+  installFlags = [
+    "DESTDIR=$(out)"
+    "prefix="
+  ];
```

`DESTDIR` should not be used, see https://github.com/NixOS/nixpkgs/issues/65718.

```suggestion
  makeFlags = [
    "prefix=${placeholder "out"}"
  ];

  installFlags = [
    "sysconfdir=${placeholder "out"}/etc"
  ];
```